systemtest/src/main/java/io/strimzi/systemtest/AbstractResources.java

```diff
@@ -63,9 +64,23 @@ KubeClient client() {
         return customResourcesWithCascading(Kafka.class, KafkaList.class, DoneableKafka.class);
     }
 
+    @SuppressWarnings("checkstyle:Regexp")
```

Comment:
What regexp is that?

Reply:
I tried to rebuild without this suppression and everything seems to be ok.